internal/proto/controller/api/resources/authtokens/v1/authtoken.proto

```diff
@@ -10,38 +10,38 @@ import "controller/api/resources/scopes/v1/scope.proto";
 // AuthToken contains all fields related to an Auth Token resource
 message AuthToken {
 	// Output only. The ID of the Auth Token.
-	string id = 10;
+	string id = 10;  // @gotags: `class:"public"`
 
 	// The Scope in which this Auth Token was generated.
-	string scope_id = 20 [json_name="scope_id"];
+	string scope_id = 20 [json_name="scope_id"];  // @gotags: `class:"public"`
 
 	// Output only. Scope information for this resource.
-	resources.scopes.v1.ScopeInfo scope = 30;
+	resources.scopes.v1.ScopeInfo scope = 30;  // @gotags: `class:"public"`
 
 	// Output only. The token value, which will only be populated after authentication and is only ever visible to the end user whose login request resulted in this Auth Token being created.
-	string token = 40;
+	string token = 40;  // @gotags: `class:"secret"`
 
 	// Output only. The ID of the User associated with this Auth Token.
-	string user_id = 50 [json_name="user_id"];
+	string user_id = 50 [json_name="user_id"];  // @gotags: `class:"public"`
 
 	// Output only. The ID of the Auth Method associated with this Auth Token.
-	string auth_method_id = 60 [json_name="auth_method_id"];
+	string auth_method_id = 60 [json_name="auth_method_id"];  // @gotags: `class:"public"`
 
 	// Output only. The ID of the Account associated with this Auth Token.
-	string account_id = 70 [json_name="account_id"];
+	string account_id = 70 [json_name="account_id"];  // @gotags: `class:"public"`
 
 	// Output only. The time this resource was created.
-	google.protobuf.Timestamp created_time = 80 [json_name="created_time"];
+	google.protobuf.Timestamp created_time = 80 [json_name="created_time"];  // @gotags: `class:"public"`
```

Comment:
Same questions as in #2004 around proto standard types.

Reply:
Will keep these around, even though they don't have any effect, to make it clear that this field is public